ggml-quants.c

```diff
@@ -13139,7 +13139,7 @@ static int iq1_find_best_neighbour(const uint16_t * restrict neighbours, const u
         const float * restrict xval, const float * restrict weight, float * scale, int8_t * restrict L, int ngrid) {
     int num_neighbors = neighbours[0];
     GGML_ASSERT(num_neighbors > 0);
-    float best_score = 0;
+    float best_score = -FLT_MAX;
```

Comment:
This function isn't used any more, but why negative FLT_MAX?

Reply:
Any negative value works, but used `-FLT_MAX` for consistency with usage of `FLT_MAX` elsewhere in the source